katas/square-digits/samuel/squareDigits.js

```diff
@@ -1,3 +1,17 @@
 export function squareDigits(num) {
-    return 0;
+    let numero = num.toString();
```

Comment:
Please use english as default language for names. In this case, use `number` instead of `numero`